scripts/release_scripts/initial_release_prep.py

```diff
@@ -0,0 +1,321 @@
+        'the job to me and the job authors: %s\n\n'
+        'Thanks!\n') % (
+            job_names_with_instructions_and_author, author_names_and_mail_ids)
+    return mail_message_template
+
+
+def get_extra_jobs_due_to_schema_changes(
+        remote_alias, previous_release_version):
+    """Finds additional jobs which should be run based on
+    schema changes in feconf.
+
+    Args:
+        remote_alias: str. The alias for Oppia repo.
+        previous_release_version: str. The version of the previous release.
+
+    Returns:
+        list(str). The list of jobs to run based on schema changes.
+    """
+    schema_versions_to_jobs_mapping = {
+        'CURRENT_COLLECTION_SCHEMA_VERSION': 'CollectionMigrationOneOffJob',
+        'CURRENT_STATE_SCHEMA_VERSION': 'ExplorationMigrationJobManager',
+        'CURRENT_SKILL_CONTENTS_SCHEMA_VERSION': 'SkillMigrationOneOffJob',
+        'CURRENT_MISCONCEPTIONS_SCHEMA_VERSION': 'SkillMigrationOneOffJob',
+        'CURRENT_RUBRIC_SCHEMA_VERSION': 'SkillMigrationOneOffJob',
+        'CURRENT_STORY_CONTENTS_SCHEMA_VERSION': 'StoryMigrationOneOffJob',
+        'CURRENT_SUBTOPIC_SCHEMA_VERSION': 'TopicMigrationOneOffJob',
+        'CURRENT_STORY_REFERENCE_SCHEMA_VERSION': 'TopicMigrationOneOffJob'}
+
+    diff_output = common.run_cmd([
+        'git', 'diff', '%s/develop' % remote_alias,
+        '%s/release-%s' % (remote_alias, previous_release_version),
+        'feconf.py'])
+    feconf_diff = diff_output[:-1].split('\n')
+
+    jobs_to_run = []
+    for version_key in schema_versions_to_jobs_mapping:
+        for line in feconf_diff:
+            if line.startswith(('+%s' % version_key, '-%s' % version_key)):
+                jobs_to_run.append(schema_versions_to_jobs_mapping[version_key])
+
+    return list(set(jobs_to_run))
+
+
+def did_supported_audio_languages_change(
+        remote_alias, previous_release_version):
+    """Checks changes in constants.SUPPORTED_AUDIO_LANGUAGES between
+    the current and previous release.
+
+    Args:
+        remote_alias: str. The alias for Oppia repo.
+        previous_release_version: str. The version of the previous release.
+
+    Returns:
+        bool. Whether supported audio languages have changed.
+    """
+    try:
+        from constants import constants
+        supported_audio_languages_for_current_release = constants[
+            'SUPPORTED_AUDIO_LANGUAGES']
+
+        common.run_cmd([
+            'git', 'checkout',
+            '%s/release-%s' % (remote_alias, previous_release_version),
+            '--', 'assets/constants.ts'])
+        from constants import constants
+        supported_audio_languages_for_previous_release = constants[
+            'SUPPORTED_AUDIO_LANGUAGES']
+    finally:
+        common.run_cmd(['git', 'reset', 'assets/constants.ts'])
+        common.run_cmd(['git', 'checkout', '--', 'assets/constants.ts'])
+
+    return (
+        sorted(supported_audio_languages_for_current_release) != sorted(
```

Comment:
Can we just check the `id` of each item in the list instead of equating the whole dict?

(We need to run the job only if a new id is added or deleted.)

Reply:
Done.